README.md

```diff
@@ -312,6 +312,9 @@ Tips for Fixture Loading Tests
         cache_sqlite_db: true
     ```
 
+    Note: You have to clear the cache every time you modify an entity or a fixture:
```

Comment:
This is not true. It happens when you modify a fixture which is only referenced by the main fixture you pass to the `loadFixtures` method.